soc/renesas/ra/ra4m1/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 
 zephyr_include_directories(.)
 
-zephyr_library_sources_ifdef(CONFIG_SOC_OPTION_SETTING_MEMORY
+zephyr_library_sources(
```

Comment:
minor nit, not blocking.

No need for linewrap anymore.

Reply:
Thank you for your feedback.
I fixed it.